packages/castmill/lib/castmill_web/components/layouts/admin.html.heex

```diff
@@ -8,7 +8,6 @@
       <.live_component
         module={CastmillWeb.Live.Admin.Sidebar}
         id={:sidebar}
-        selected_link={@selected_link}
```

Comment:
How will the selected link be activated then?

Reply:
Nothing is selected in the sidebar since we are at the admin base route.